01-html/05-aol.html

```diff
@@ -2,4 +2,130 @@
 I picked a signup form for something that I (hope) you don't already have an account for.
 
 Forms are important!
--->
\ No newline at end of file
+-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AOL | Sign Up</title>
+  </head>
+<body>
+  <header>
+    <h4>Already have an account?<a href="https://my.screenname.aol.com/">   Sign In</a>
+      <a href="https://help.aol.com/"> | Help</a>
+    </h4>
+  </header>
+  <aside>
+    <section>
+      <h1><b>AOL</b> Mail.</h1>
+      <h1>So many reasons to love AOL Mail</h1>
+    </section>
+    <section>
+      <h2>You're covered</h2>
+      <h4>Unlimited storage, 25MB of photo <br>and video attachments, advanced spam <br>filters, virus protection - all as part of <br>your Inbox.</h4>
+      <img src="http://thumb101.shutterstock.com/photos/display_pic_with_logo/2390648/208349443.jpg" alt="Mail Icon" height="25" width="25">
+      <img src="https://www.presentationpro.com/images/product/medium/slide/PPP_IFlat_LT3_Flat_Paper_Clip_01_Circle.jpg" alt="Email Attachment Icon" height="25" width="25">
+      <img src="https://image.freepik.com/free-icon/inbox--appbar_318-32164.png" alt="Inbox Icon" height="25" width="25">
+    </section>
+    <section>
+      <h2>Stay connected wherever you go</h2>
+      <h4>Connect on mail, chat or text using your <br>favorite laptop, tablet or mobile device.</h4>
+      <img src="https://cdn2.iconfinder.com/data/icons/modern-future-technology-2/128/apple-devices-2-128.png" alt="Device Icon" height="100" width="100">
+    </section>
+    <section>
+      <h2>More than just mail</h2>
+      <h4>Right from your Inbox - get direct access <br>to the latest headlines and productivity <br>tools to make life simpler.</h4>
+      <img src="http://spreadsheetsapp.com/img/huffington_post.png" alt="Huffington Post Icon" height="50" width="100">
+      <br>
+      <img src="http://vignette1.wikia.nocookie.net/logopedia/images/b/b2/AIM.png/revision/latest/scale-to-width-down/250?cb=20140920233828" alt="aim logo" height="40" width="60">
+    </section>
+  </aside>
+  <form action="https://i.aol.com/reg/signup" method="post">
+    <fieldset>
+      <h2>Let's create your account</h2>
+      <p><label for="name">Name</label>
+        <br>
+        <input type="text" name="name" placeholder="First Name" size="22" minlength="2" maxlength="50" required>
+        <input type="text" name="name" placeholder="Last Name" size="25" minlength="2" maxlength="50" required>
+      </p>
+      <p><label for="username">Pick a username</label>
+        <br>
+        <input type="text" name="username" placeholder="@aol.com" size="50" minlength="3" maxlength="100" required>
+      </p>
```

Comment:
A good practice for forms is to wrap your `input` tag in your `label` tag. It doesn't quite make intuitive sense, but it improves usability. By doing so, here's what you would gain: clicking on "Pick a username", for example, will cause the  cursor to focus on the input field. They aren't using it on the name field, but try it on any other fields on the site.